price-estimator/src/models.rs

```diff
@@ -79,11 +79,25 @@ pub struct MarketsResult {
     pub bids: Vec<TransitiveOrder>,
 }
 
+/// Sort orders by their price and merge orders with the same price into one.
+fn sanitize_orders(mut orders: Vec<TransitiveOrder>) -> Vec<TransitiveOrder> {
```

Comment:
nit: I think this should be part of the pricegraph API, specifically `ask_prices` and `bid_prices`.